package/MDAnalysis/lib/nsgrid.pyx

```diff
@@ -69,249 +70,52 @@ not reflect in the results.
 .. [#] a pair correspond to two particles that are considered as neighbors .
 
 .. versionadded:: 0.19.0
+.. versionchanged:: 2.0.0
```

Comment:
Can't this go in 1.0.2 ?

Reply:
Yeah no reason why not, I think this is now @IAlibay 's call